quickwit/quickwit-doc-mapper/src/default_doc_mapper/date_time_type.rs

```diff
@@ -301,7 +301,7 @@ mod tests {
                 "description": "When the record was last updated.",
                 "input_formats": ["iso8601"],
                 "output_format": "rfc3339",
-                "precision": "seconds",
+                "precision": "second",
```

Comment:
do we have backward compatibility?

We also need to update all examples in the code.

Reply:
| do we have backward compatibility?

Do you mean in tantivy or in the metastore? tantivy accepts both currently

Reply:
We have backward compatibility. I think we should revert that change, but in case Adrien has a good case for it I opened two PRs to fix the doc and simplify the tantivy stuff.